_data/simple-icons.json

```diff
@@ -8918,7 +8918,7 @@
         },
         {
             "title": "OpenJDK",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

Comment:
Not sure black is really representative for this icon. I think we could probably use `#437291` or `#E76f00`.
What do you think?

Reply:
I think black is better. It's head and hands are in black.

Reply:
The blue and orange are both prominently used [on their GitHub profile](https://github.com/openjdk) alongside the black/white/red mascot. Personally, I'd go with `#437291` given it's used more in the GitHub profile picture - and is the bolded text in the text logo.

Reply:
I don't think we can tamper with the skin color of their mascot, which is black.

Reply:
Much like @service-paradis though, I see the hex we provide as a representation of the brand, more than a representation of the mascot that accompanies it. Given the blue and orange are used prominently (albeit behind the mascot in some) I would say either of those is the 'brand colour' over black. The mascot will _display_ as white/black in our previewer anyway!

Reply:
This is the pale blue for sure given how ["JDK"](https://openjdk.org/groups/build/doc/building.html) is written in it, too!